### PR TITLE
Monkeypatch fix

### PR DESCRIPTION
**Pull request by tschuy**
This allows the monkeypatch to work with both projects by using the resource name rather than a hardcoded name (as before) or an attribute (which would be parsed only once, and work for only the first project).

### CODE REVIEW

File: recipes/_monkey_patch.rb [hunk 1]
```diff
@@ -64,7 +64,7 @@ def action_before_deploy
             environment new_resource.environment
           end
           django_resource = new_resource.application.sub_resources.select{|res| res.type == :django}.first
-          base_command = "#{::File.join(django_resource.virtualenv, "bin", "gunicorn")} whats_fresh.wsgi:application"
+          base_command = "#{::File.join(django_resource.virtualenv, "bin", "gunicorn")} #{new_resource.application.name}.wsgi:application"
```
Comment (jordane) [inline recipes/_monkey_patch.rb]: Please document that this name now matters somewhere.

Comment (tschuy) [inline recipes/_monkey_patch.rb]: It's always mattered, the application cookbook uses it for things like this I believe. I'll add to the README.

Comment (jordane) [inline recipes/_monkey_patch.rb]: I suspected as much. Thanks.

File: attributes/default.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 default['whats_fresh']['server_name'] = node['fqdn']
 default['whats_fresh']['gunicorn_port'] = 8080
+default['whats_fresh']['access_log'] = '/var/log/nginx/whats_fresh/access.log'
+default['whats_fresh']['error_log'] = '/var/log/nginx/whats_fresh/error.log'
```
Comment (armiller) [inline attributes/default.rb]: If we are using the nginx cookbook this should be:
```Ruby
"#{node['nginx']['log_dir']}/whats_fresh/error.log"
```

File: metadata.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description "Installs/Configures What's Fresh dependencies"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.1.11'
+version '0.1.15'
```
Comment (jordane) [inline metadata.rb]: We don't do version bumps in PRs.

Comment (tschuy) [inline metadata.rb]: The reason that the version is bumped is because I had to push this to the chef server already to deploy Working Waterfronts.

**Comment by iankronquist** [comment]
This will be unnecessary once we use the python-webapp cookbook and #11 is resolved.

File: metadata.rb (hunk 3 above)
Comment (ramereth) [inline metadata.rb]: So is this tree what's currently uploaded to our chef server? I'm trying to work on some other cleanup on this cookbook and I want to make sure I'm working with the correct version.

Comment (tschuy) [inline metadata.rb]: Yes, it is.

Comment (ramereth) [inline metadata.rb]: @jordane any objection if we just go ahead and merge this since its already in production?

Comment (jordane) [inline metadata.rb]: +1

**Pull request merged**